src/posts/2021-06-14-Everyday_Accessibility.md

```diff
@@ -0,0 +1,147 @@
+	<figcaption><b>Key Point:</b> Microsoft's "Check Accessibility" feature helps ensure your document is inclusive.</figcaption>
+</figure>
+
+## Audio and Video Content
+
+We can all create multimedia, but let's ensure we don't ignore the best practices.
+
+### Have clear audio
+
+Though all of our devices can record audio and video on-demand, sound quality can vary making it difficult for some to hear the content.  Be sure to speak as clearly as possible, preferably at a steady pace and reasonable volume, and remove or avoid background noise including echo, empty room effect, humming, or white noise.
+
+**Key Point:** Using a microphone (even a common corded headset) can help amplify voices and minimize distracting unwanted sounds.
+
+### Reduce background music
+
+Instrumental music can enhance content’s quality, but not when it reduces the ability to hear the main speaker. When using background music, keep it to a minimum (such as during an intro) or, if used throughout, ensure it is very low.
+
+**Key Point:** Consider putting your background music at the volume of a whisper.
+
+### Provide text with multimedia
+
+As with images, video and audio can be engaging but also inaccessible without a text equivalent for people who are deaf-blind, having sensory processing disorders, or have hearing impairments. Having media content written out as text is vital for inclusiveness. Text equivalents can include transcripts that type out spoken dialogue, or an article that accompanies a video (as in a video that supplements a news article).
+
+**Key Point:** Where there’s media, make sure there’s also text.
+
+### Avoid flashing content
+
+Content that flashes on the screen more than 3 times per second is dangerous as it can trigger a seizure in some viewers. Use rapid movement sparingly in your videos and ensure nothing flashes. You can use the Trace Center’s [Photosensitive Epilepsy Analysis Tool (PEAT)](https://trace.umd.edu/peat/) to identify seizure risks in web content and software.
+
+**Key Point:** Creating content comes with a responsibility to [create safe content](https://www.w3.org/WAI/WCAG21/Understanding/three-flashes).
+
+## Advocate for Accessibility
+
+We have to work collectively to make inclusion happen.
+
+### Seek accessibility features
+
+Actively search for “accessibility” in the tools you use every day — from survey creation sites to website design platforms. Understand what you can do to make sure your output is inclusive.
+
+<figure role="figure" aria-label="Search for accessibility tips in your favorite systems like on the Zoom accessibility page.">
+	<img alt="Zoom video conferencing Accessibility landing page showing their accessibility features." src="/img/posts/2021-06-14-Everyday_Accessibility/Zoom_A11y_Page.png" />
```

Comment:
The preview is showing a broken image here. I think we need to delete `Accessible_Typeforms.png `, and then add `Zoom_A11y_Page.png` so it loads properly.

Reply:
Yay! Yes, I think I emailed that to @mxmason because I don't have access/permissions to upload. If you want to message me a place to email it I can do that ASAP so we can get rolling! This was a great collaboration!

Reply:
Just pushed it up!